deeplearning-models/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+set( TARGET deeplearning-models )
+set( TARGET_ROOT ${topdir}/deeplearning-models )
+
+message( STATUS "========== Configuring ${TARGET} ==========" )
+
+def_vars()
+
+list( APPEND SOURCES
```

Comment:
Better to write `set( SOURCES`, in case the variable SOURCES inherits some value from parent scope that woud influence the CMake evaluation incorrectly.

Reply:
resolved in 7e642c2d8da630073e45fe99d4e9687131e97a22